src/data/memorialPlacesData.ts

```diff
@@ -0,0 +1,65 @@
+export interface Place {
```

Comment:
If it is gonna be used across project move it to some common `types` module.
If not, define it in the Props of the consuming component.